### PR TITLE
VIH-9446 - play notification sounds using base64 values

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-9446

### Change description ###

play notification sounds using base64 values

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by will-craig — DISMISSED** [review]
Changing the mp3 to a base64 text file was smart. Wouldn't have thought of that. Nice.

File: VideoWeb/.vscode/settings.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
   "editor.rulers": [120, 140],
   "dotnet-test-explorer.testProjectPath": "**/*[Tt]ests/*.csproj",
   "markiscodecoverage.searchCriteria": "**/lcov*.info",
-  "prettier.configPath": ".prettierrc"
```
Comment (will-craig) [inline VideoWeb/.vscode/settings.json:6]: I don't think .vscode shouldbe included in the repo. It should be in the .gitignore

Comment (shaed-parkar) [reply VideoWeb/.vscode/settings.json:6]: I would agree but it's in the repo and it's wrong so I figured I should fix it

**Comment by shaed-parkar** [comment]
> Changing the mp3 to a base64 text file was smart. Wouldn't have thought of that. Nice.

It's the power of strings 😄

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-sounds.service.ts [hunk 2]
```diff
@@ -10,12 +10,12 @@ export class NotificationSoundsService {
     hearingStartingAnnounced: boolean;
     currentPlayCount: number;
     hearingAlertSound: HTMLAudioElement;
-
     hearingAlertPlayCount: number;
 
-    initConsultationRequestRingtone(): void {
+    async initConsultationRequestRingtone() {
```
Comment (marcogagliardi) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-sounds.service.ts:15]: is there any reason to remove the return signature? Just curious.

Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/notification-sounds.service.ts:15]: I noticed this was the only one to have it in the method signature. TypeScript also makes it easy to infer what the returning type is

**Review by will-craig — APPROVED** [review]

**Pull request merged**